shortfin/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ add_compile_options("$<$<C_COMPILER_ID:MSVC>:/utf-8>")
 add_compile_options("$<$<CXX_COMPILER_ID:MSVC>:/utf-8>")
 
 # Pins
-set(SHORTFIN_IREE_GIT_TAG "iree-3.1.0rc20241220")
+set(SHORTFIN_IREE_GIT_TAG "iree-3.1.0rc20241204")
```

Comment:
Can't tell if this hurts or helps. One workflow run at least seemed to stall: https://github.com/nod-ai/shark-ai/actions/runs/12637987386/job/35213328233?pr=761

Reply:
Seems to be stalling again on a re-run: https://github.com/nod-ai/shark-ai/actions/runs/12637987386/job/35215801288?pr=761